Userland/Utilities/aplay.cpp

```diff
@@ -42,6 +42,10 @@ int main(int argc, char** argv)
             out("{}/{}", loader->loaded_samples(), loader->total_samples());
             fflush(stdout);
             audio_client->enqueue(*samples);
+        } else if (loader->has_error()) {
+            outln();
+            out("Error: {}", loader->error_string());
```

Comment:
This is weird. Why print a newline *before* the error message, but not after?

Reply:
The first newline is necessary because the progress bar stays on the same line in order to replace itself. I just forgot about a newline after.